packages/gatsby-plugin-onesignal/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "@vtex/gatsby-plugin-onesignal",
+  "version": "0.0.0",
+  "description": "Gatsby plugin for a simple OneSignal use case.",
+  "main": "index.js",
+  "scripts": {
+    "build": "tsc",
+    "prepare": "cross-env NODE_ENV=production yarn run build",
+    "develop": "tsc --watch"
+  },
+  "keywords": [
+    "gatsby",
+    "gatsby-plugin",
+    "onesignal",
+    "vtex"
+  ],
+  "license": "MIT",
+  "peerDependencies": {
+    "gatsby": "^3.1.1",
+    "react": "^0.0.0-experimental-7f28234f8",
+    "react-dom": "^0.0.0-experimental-7f28234f8"
```

Comment:
do you need concurrent mode?

Reply:
Nope! Nice catch, I actually forgot to update this 😅